app/partials/referencedByView.html

```diff
@@ -0,0 +1,30 @@
+<record-address-bar ng-if="showAddressBar"></record-address-bar>
+<!-- TODO: search bar disabled for alpha because not coded yet -->
+<search-bar ng-if="false" ng-controller="recordSearchController"></search-bar>
+<div ng-if="record" class="column-wrapper">
+    <div class="tree-columns">
+        <span ng-repeat-start="column in columns" class="column column-{{::$index}}" ng-class="{'sortable' : column.canSort}" ng-click="toggleSort(column)">
+            {{::column.label}}
+            <span class="sort-arrow" ng-if="column.label === sort.column" title="Elements are sorted by this column in {{sort.reverse ? 'ascending' : 'descending'}} order.">
+                <i class="fa fa-arrow-{{sort.reverse ? 'up' : 'down'}}"></i>
+            </span>
+        </span>
+        <split-bar ng-repeat-end ng-if="!$last" offset="16" resize-callback="columnResized"></split-bar>
+        <div class="column-action edit-columns" ng-click="openColumnsModal()" title="Edit columns"></div>
```

Comment:
Is there any need to support custom columns in this view?  Maybe?  Can we think of a potentially useful custom column?

Reply:
Per my later comment, I do think having custom columns is sensible here.